EOLib/Domain/Protocol/InitReply.cs

```diff
@@ -5,14 +5,15 @@ public enum InitReply
         ClientOutOfDate = 1,
         Success = 2,
         BannedFromServer = 3,
-        MapFile = 4,
```

Comment:
Is this enum just wrong? I grabbed it from eoserv years ago, and have wondered why some of the packets used a `Char` for these while some used `Byte`, but that would explain it

Reply:
That’s was my understanding from testing against GameServer. Eoserv actually adds the banned reply (as a chat) for warp map downloads haha

Reply:
Haha yeah I saw that when I checked it out...good find on this, it's a lot cleaner this way